.circleci/config.yml

```diff
@@ -2,7 +2,7 @@
 # syntax, such as anchors, will be fixed automatically.
 version: 2.1
 orbs:
-  shared: getoutreach/shared@2.17.0
+  shared: getoutreach/shared@dev:first
```

Comment:
Is this correct?

Reply:
Yep, required right now. There's a note on https://github.com/getoutreach/delibird/pull/48#discussion_r1294912863. Copied for posterity:

> Correct, it'll cause issues in 90 days. It's required right now, unfortunately. Once we release on Monday, we can bump this all to rc and get off of the dev:first orb.